packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -55,10 +55,10 @@ contract LongShortPair is Testable, Lockable {
     enum ContractState { Open, ExpiredPriceRequested, ExpiredPriceReceived }
     ContractState public contractState;
 
-    string public pairName;
-
     uint64 public expirationTimestamp;
 
+    string public pairName;
```

Comment:
I know the suggestion was to not split up `contractState` and `expirationTimeestamp` but for the sake of future maintainers, I'd like to still include a @dev comment here explaining why `pairName` needs to follow `expirationTimestamp` which is a uint64

Reply:
Good point! +1